pwiz_tools/Skyline/TestData/LocalizedResourcesTest.cs

```diff
@@ -91,24 +109,34 @@ private void ValidateLocalizedResource(object invariantValue, object localizedVa
             {
                 return;
             }
+
             Assert.AreEqual(invariantValue?.GetType(), localizedValue.GetType(), message);
             var invariantText = invariantValue as string;
             if (invariantText != null)
             {
                 var localizedText = (string)localizedValue;
                 Assert.AreEqual(MaxSubstitutionIndex(invariantText), MaxSubstitutionIndex(localizedText), message);
                 StringAssert.DoesNotMatch(localizedText, RegexLowercaseMnemonic, "Mnemonic should be uppercase: {0}", message);
+                Assert.IsFalse(localizedText.StartsWith(ENGLISH_ERROR_PREFIX), "Localized text should not start with the English '{0}' {1}", ENGLISH_ERROR_PREFIX, message);
+                if (ErrorChecker.IsErrorLine(invariantText))
+                {
+                    Assert.IsTrue(ErrorChecker.IsErrorLine(localizedText), "Localized text should start with localized form of 'Error:' {0}", message);
```

Comment:
This seems like a subtle error message for the current case where a colon and space were left in place instead of using the Unicode double-wide colon and no space.

Reply:
Also, I think you probably want to explicitly check the colon, because to an untrained eye, it is difficult to notice the difference between a Unicode double-wide colon and an ascii colon with a space.